src/CIQS-CLI-Extension/azext_ciqs/format.py

```diff
@@ -0,0 +1,41 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from collections import OrderedDict
+from collections import defaultdict
+
+def transform_templateListItem(result):
+    result = OrderedDict([('Template Id', result['id']),
+                        ('Title', result['title']),
+                        ('Category', result['category'])])
```

Comment:
Can I return custom field like ('Deployment Link', UX_BASE_URL + '/Deployments/new/' + result['id']).

I would love to have that for "Deployment Link' and 'Try It Now'.